github/client.go

```diff
@@ -873,7 +873,7 @@ func (client *Client) FindOrCreateToken(user, password, twoFactorCode string) (t
 	}
 
 	params := map[string]interface{}{
-		"scopes":   []string{"repo"},
+		"scopes":   []string{"repo", "gist"},
```

Comment:
What about current hub users who already have authenticated and have a token, but their token doesn't have the "gist" scope? For them, all calls to `hub gist` will fail with an HTTP error. Maybe we could handle that error and suggests that they open `github.com/settings/token` and edit the scopes to include "gist".

Reply:
I'd love to do that! As you probably saw I spent a lot of time not able to understand why this wasn't working and thought the API was broken. I'll figure out how to do that.